assets/data/effects/drauven/drauven_pc_drauven_guerilla_upgrade_movement.json

```diff
@@ -0,0 +1,47 @@
+{
+"id": "drauven_pc_drauven_guerilla_upgrade_movement",
+"info": {
+	"dataVersion": 1,
+	"sourceFile": "drauven/drauven_pc_drauven_guerilla_upgrade_movement",
+	"author": "Bipen",
+	"tags": [ "okToMarkDone", "sneaky" ]
+},
+"type": "ABILITY",
+"verb": "MANEUVER",
+"ability": {
+	"icon": "zealousLeap",
+	"category": "research",
+	"priority": "1",
+	"showInCharacterSheet": false
+},
+"targets": [
+	{
+		"template": "SELF",
+		"choose": "BY_SCORE",
+		"scoreFunction": "1-min(1,pinned)",
+		"scoreThreshold": "1",
+		"missionFeedback": { "orientTo": "targetTile" }
+	},
+	{
+		"role": "targetTile",
+		"template": "TILE",
+		"missionFeedback": {
+			"rangeFeedback": "FRIENDLY_EFFECT_RANGE",
+			"suggestionFeedback": "FRIENDLY_EFFECT_SUGGESTION",
+			"hoverFeedback": "PATH_TARGET",
+			"showMovementFeedback": true
+		},
+		"minRange": "0.9",
+		"range": "max(floor((self.SPEED)),1.6)",
+		"lineOfSight": "TERRAIN_ONLY",
+		"tileFilter": "validMoveEnd"
+	}
+],
+"outcomes": [
+	{ "class": "MoveEntity", "directlyToTile": "targetTile" },
```

Comment:
Right now this is causing the character to jump straight to the target tile. You might want to look at `heroicDash.json` (in `<base_game>/assets/data/effects/heroism`) for an example of movement that makes the character walk along the path to the target tile

Reply:
this has been resolved now it gives you a free move action instead of the whirl type dash... same distance as your speed stat